### PR TITLE
Update escalus to newest version

### PR DESCRIPTION
**Pull request by bszaf**

### CODE REVIEW

File: relx.config [hunk 1]
```diff
@@ -8,7 +8,6 @@
   {exml, load},
```
Comment (mentels) [inline relx.config]: Why do we get rid of these dependencies?

**Comment by mentels** [comment]
And the Travis failed..

File: relx.config (hunk 1 above)
Comment (bszaf) [reply relx.config]: Release will not be build otherwise. It seems that deps of deps were added into `relx.config`

**Comment by michalwski** [comment]
This was superseded by #60

**Pull request closed without merging**